draft-ietf-quic-transport.md

```diff
@@ -907,9 +896,9 @@ When the client receives a Version Negotiation packet from the server, it should
 select an acceptable protocol version.  If the server lists an acceptable
 version, the client selects that version and reattempts to create a connection
 using that version.  Though the contents of a packet might not change in
-response to version negotiation, a client MUST increase the packet number it
-uses on every packet it sends.  Packets MUST continue to use long headers and
-MUST include the new negotiated protocol version.
+response to version negotiation, a client MUST increase the packet number by one
```

Comment:
This requirement of increment by 1 is broken around migration. This should probably make an exception for that, with a pointer to Section 7.6... but that's probably an overkill. How about "a client MUST increase the packet number by one for every packet it sends in response to version negotiation."

Reply:
Right, though you won't be migrating during version negotiation, so your fix works.